### ISSUE TITLE
Modification time of directory is changed to current time after viewing images in the directory

### ISSUE THREAD

**Issue by StevenKuhnert** [d2phap/ImageGlass#1903]
### System information:
 - Windows OS version: Windows 10 Pro 22H2
 - ImageGlass version: ImageGlass v9.0.11.502

### To Reproduce
1. Open an image in any directory
2. Modification time of directory is changed to current time

### Actual behavior:
Modification time of directory is changed to current time after viewing images in the directory since version update from 9.0.10.201 to 9.0.11.502.
Possibly caused by added feature "OpenSaveAsDialogInTheCurrentImageDir"? But changing flag from true to false does not change anything.

### Expected behavior:
Modification time of directory should not be changed like in version 9.0.10.201

### Additional context:
I have to switch back to version 9.0.10.201 to not mess up my directory structure.

**Comment by d2phap** [comment]
@StevenKuhnert 
I cannot reproduce this. Did you do any actions like copy, paste, rename, delete...?

**Comment by StevenKuhnert** [comment]
No, the problem occures without taking any actions like copy, paste, rename, delete... Only open an image in any directory from Windows Explorer changes the directory modification time to current time of the directory from where the image is displayed.

I've just installed new version 9.1.8.723 (64-bit) and this problem is gone! 

When changing back to version 9.0.11.502 the problem occures again, so obviously the bug is fixed with the new version!?
